packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -135,9 +189,14 @@ function ShadowItem( { shadow, category } ) {
 	return (
 		<NavigationButtonAsItem
 			path={ `/shadows/edit/${ category }/${ shadow.slug }` }
-			icon={ shadowIcon }
 		>
-			{ shadow.name }
+			<HStack justify="space-between">
+				<HStack justify="flex-start">
+					<IconWithCurrentColor icon={ shadowIcon } />
+					<FlexItem> { shadow.name } </FlexItem>
+				</HStack>
+				<IconWithCurrentColor icon={ chevronRight } />
```

Comment:
It looks like the chevrons were added in https://github.com/WordPress/gutenberg/pull/67720. We should merge trunk into this PR and resolve any conflicts.

Reply:
I believe the approach I used to add the chevrons is better suited to accommodate the changes proposed in this PR. I’d like to continue with the same method, but please let me know if you have suggestions for a better approach.

Reply:
Could you explain the approach you're referring to here? At the moment it looks like this PR is using the same method to add the chevrons:

`<Icon icon={ isRTL() ? chevronLeft : chevronRight } />`

Reply:
I was referring to the extra `Hstack`, however now as we have decided to remove it lets ignore that 😇